src/Tizen.NUI/src/public/Utility/RenderEffect.cs

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright(c) 2024 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+using System;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// View's optional render effect.
+    /// Applications can apply RenderEffect as the example below :
+    /// <code>
+    /// 
+    /// RenderEffect effect = RenderEffect.CreateBackgroundBlurEffect(20.0f);
+    ///
+    /// view.SetRenderEffect(effect);
+    /// view.ClearRenderEffect();
+    ///
+    /// </code>
+    /// Note that a view owns at most one render effect.
+    /// </summary>
+    /// <since_tizen> 12 </since_tizen>
```

Comment:
```suggestion
     [EditorBrowsable(EditorBrowsableState.Never)]
```

Please make it as hidden